application/client-test/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "tests",
+  "version": "1.0.0",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "clean": "oasis clean",
+    "build": "oasis build",
+    "lint": "eslint *.js test/ config.js",
+    "lint:fix": "eslint *.js test/ config.js --fix",
+    "test-local": "env CHAIN=local mocha",
+    "test-devnet": "env CHAIN=devnet mocha"
+  },
+  "devDependencies": {
+    "bip39": "^2.5.0",
+    "command-line-args": "^5.1.1",
+    "eslint": "^5.12.1",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "mocha": "^6.1.4"
+  },
+  "dependencies": {
+    "@oasislabs/client": "^1.0.0-rc.3"
```

Comment:
@armaniferrante is this current? I thought it was on 4 or something now.

Reply:
The current can be found on the README https://github.com/oasislabs/oasis.js. Currently we're at 6.